esa/saw.py

```diff
@@ -1431,6 +1446,35 @@ def SolvePowerFlow(self, SolMethod: str = 'RECTNEWT') -> None:
         script_command = "SolvePowerFlow(%s)" % SolMethod.upper()
         return self.RunScriptCommand(script_command)
 
+    def OpenOneLine(self, filename, view="", fullscreen="NO", showfull="NO"):
```

Comment:
We should include all allowed parameters if we're wrapping a ScriptCommand. I'm making this update now.